### ISSUE TITLE
User-Translations + Anzeige Textbox

### ISSUE THREAD

**Issue by andneuma** [kjoscha/lichtenberg#35]
Nicht prioritär!

Momentan ists ja so, dass in der Description-Box eines POIs ein leeres Feld steht, sofern da keine Beschreibung für die entsprechende am Anfang ausgewählte Sprache eingetragen ist. Es wäre vielleicht cool, da sowas wie 'Für diesen Beschreibungstext liegt keine Übersetzung vor' (jeweils dann in der ausgewählten Sprache natürlich). Sonst könnte man auf die Idee kommen, dass die Seite spinnt.

**Comment by kjoscha** [comment]
finde ich schon wichtig. ist auch ne Sache von 5 Minuten - mit direkter Verlinkung zum Editor etwas länger

**Comment by kjoscha** [comment]
erste Version ist online...

**Comment by kjoscha** [comment]
mal wieder cache erneuern. das ist auch ein todo ;)

**Comment by andneuma** [comment]
Jo, kann man als Farbe dann rot nehmen? Auf französisch würde das glaube ich "Pas encore de traduction française" heißen. Kann man das sprach-versions-spezifisch da anzeigen lassen?

**Comment by kjoscha** [comment]
ja & ja

**Comment by kjoscha** [comment]
so, farbe und verlinkung drin
